cypress/components/popover-container/popover-container.cy.tsx

```diff
@@ -253,66 +290,76 @@ context("Test for Popover Container component", () => {
   });
 
   describe("check events for Popover Container component", () => {
-    let callback;
-
-    beforeEach(() => {
-      callback = cy.stub().as("callback");
-    });
-
     it("should call onOpen callback when a click event is triggered", () => {
+      const callback: PopoverContainerProps["onOpen"] = cy.stub().as("onOpen");
       CypressMountWithProviders(<PopoverContainer onOpen={callback} />);
 
       popoverSettingsIcon().click();
-      cy.get("@callback").should("have.been.calledOnce");
+      cy.get("@onOpen").should("have.been.calledOnce");
     });
 
-    it.each([["Enter"], ["Space"]])(
+    it.each([...keyToTrigger])(
```

Comment:
just to be sure that all keys are used I would change the title to use `%s` key to show that :)